### ISSUE TITLE
"Channel is not registered" running releases:retry

### ISSUE THREAD

**Issue by edmorley** [heroku/heroku-releases-retry#3]
```
$ export HEROKU_APP=treeherder-stage
$ heroku releases:retry
Retrying v1622 on treeherder-stage... done, v1623
 !    Release command declared: this new release will not be available until the command succeeds.
Running release command...
Channel is not registered.
 !    Release command starting. Use `heroku releases:output` to view the log.
```

Whilst the retried release succeeded, the release phase command output is missing from the console and the `Channel is not registered` message shown instead, which presumably means the request for it 404ed:
https://github.com/heroku/heroku-releases-retry/blob/f9770139d64bf3ef5285630249996179ae6f6a59/commands/retry.js#L38-L39

$ heroku --version
heroku-cli/6.15.24-e5de04c (win32-x64) node-v9.3.0

$ heroku plugins
heroku-api-plugin 2.1.4
heroku-buildkits 1.0.3
heroku-builds 0.0.24
heroku-releases-retry 1.0.2
heroku-slugs 1.0.3

**Comment by edmorley** [comment]
Using the command later for another app showed the release phase log fine:
```
$ heroku releases:retry
Retrying v281 on treeherder-prod... done, v282
 !    Release command declared: this new release will not be available until the command succeeds.
Running release command...
-----> PRE-DEPLOY: Running Django system checks...
System check identified no issues (1 silenced).
...
```

**Comment by dmathieu** [comment]
Yes, this is a kind of race condition that can happen when the channel hasn't been created fast enough.
We should retry a couple times on 404s like this.

**Comment by dmathieu** [comment]
We could retry on those 404s. We had a good reason for not doing so, but I don't remember what it is. I'll come back with more details.

**Comment by beccagsc** [comment]
Any update @dmathieu ?

**Comment by dmathieu** [comment]
Nope, sorry. Nothing has changed here.
This only impacts CLI output though. Release phase runs properly in those cases.

**Comment by beccagsc** [comment]
Thanks @dmathieu!
